gemma-core/pom.xml

```diff
@@ -120,8 +72,6 @@
                 <configuration>
                     <repositoryName>lib</repositoryName>
                     <repositoryLayout>flat</repositoryLayout>
-                    <!--suppress UnresolvedMavenProperty -->
-                    <extraJvmArguments>-Dlog4j.configurationFile=@BASEDIR@/etc/log4j.properties</extraJvmArguments>
```

Comment:
This never worked, the file in etc/ was picked-up because Appassembler adds it to the classpath.